R/chromote_session.R

```diff
@@ -204,10 +204,31 @@ ChromoteSession <- R6Class(
       # A data frame of targets, one row per target.
       info <- fromJSON(self$parent$url("/json"))
       path <- info$devtoolsFrontendUrl[info$id == private$target_id]
+
       if (length(path) == 0) {
         stop("Target info not found.")
       }
 
+      if (grepl("^https://chrome-devtools-frontend\\.appspot\\.com", path)) {
```

Comment:
We could also put this behind an option flag, e.g. `chromote.use_legacy_inspector_path` or something similar, that defaults to `TRUE`. That would give an out to anyone who wants to use the new URL or if the `/devtools/inspector.html` path changes or breaks in the future (there are no signs that it would). It would also give us a place to document how to set the `--remote-allow-origins` arg.